docs/reference/hacl/index.md

```diff
@@ -14,6 +14,7 @@ signature/index
 dh/index
 mac/index
 kdf/index
+randomness/index
```

Comment:
Didn't we say we wanted to advertise the evercrypt API only for now?
I'm fine with leaving this in, but then we should add a pointer to the evercrypt docs in the beginning, saying that's the one to use.

Reply:
I think we agreed on "Let's put everything in for now and see later what we want to keep."